.nlds.stylelintrc.json

```diff
@@ -0,0 +1,118 @@
+{
+  "extends": ["stylelint-config-standard-scss"],
+  "plugins": ["stylelint-order"],
+  "rules": {
+    "order/order": ["custom-properties", "declarations"],
+    "order/properties-alphabetical-order": true,
+    "scss/at-mixin-pattern": "^[a-z][a-z0-9-_]*$",
+    "scss/at-rule-no-unknown": true,
+    "scss/dollar-variable-default": true,
+    "scss/dollar-variable-first-in-block": [true, { "ignore": ["comments", "imports"] }],
+    "scss/at-extend-no-missing-placeholder": null,
+    "at-rule-no-unknown": null,
+    "block-no-empty": [true],
+    "color-no-invalid-hex": [true],
+    "comment-no-empty": [true],
+    "declaration-block-no-duplicate-properties": [true, { "ignore": ["consecutive-duplicates-with-different-values"] }],
+    "declaration-block-no-shorthand-property-overrides": [true],
+    "font-family-no-duplicate-names": [true],
+    "font-family-no-missing-generic-family-keyword": [true],
+    "function-calc-no-unspaced-operator": [true],
+    "function-linear-gradient-no-nonstandard-direction": [true],
+    "keyframe-declaration-no-important": [true],
+    "media-feature-name-no-unknown": [true],
+    "no-descending-specificity": [true],
+    "no-duplicate-at-import-rules": [true],
+    "no-duplicate-selectors": [true],
+    "no-empty-source": [true],
+    "no-invalid-double-slash-comments": [true],
+    "property-no-unknown": [true],
+    "selector-pseudo-class-no-unknown": [true],
+    "selector-pseudo-element-no-unknown": [true],
+    "string-no-newline": [true],
+    "unit-no-unknown": [true],
+    "alpha-value-notation": ["percentage"],
+    "hue-degree-notation": ["number"],
+    "color-function-notation": ["modern"],
+    "length-zero-no-unit": [
+      true,
+      {
+        "ignore": ["custom-properties"]
+      }
+    ],
+    "font-weight-notation": [
+      "numeric",
+      {
+        "ignore": ["relative"]
+      }
+    ],
+    "function-url-no-scheme-relative": [true],
+    "unit-disallowed-list": [["s"]],
+    "font-family-name-quotes": ["always-unless-keyword"],
+    "function-url-quotes": ["always"],
+    "declaration-property-value-disallowed-list": [
+      {
+        "text-align": ["left", "right"]
+      }
+    ],
+    "property-disallowed-list": [
```

Comment:
Is er ook een optie om disallowed `prop: val` aan te geven? Bijvoorbeeld `float: left` is met deze lijst nog steeds mogelijk

Reply:
Dat kan met deze rule: [declaration-property-unit-disallowed-list](https://stylelint.io/user-guide/rules/declaration-property-unit-disallowed-list)
```
    "declaration-property-unit-disallowed-list": {
      "float": ["left", "right"]
    },
```
⬆️ toegevoegd

Reply:
Ik zie alleen `"text-align": ["left", "right"]` staan op regel 53

Kun je deze aanvullen met wat er op MDN staat? 
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values#properties_for_floating

Reply:
Momenteel vind ik dit te triviaal om op te wachten. Dan heb ik liever een story op de backlog om dit later nog te doen. Gaandeweg zullen we meer dingen tegenkomen die we nog willen veranderen. Dat kunnen we dan in dat ticket verzamelen tot we het oppakken